index.html

```diff
@@ -0,0 +1,631 @@
+        </div>
+    </aside>
+
+    <section class="section-2">
+        <div class="flex dennis">
+            <a href="#">By Dennis Overbye</a>
+        </div>
+        <div class=" d-icons flex">
+            <div>
+                <span class="date">March 17, 2014</span>
+            </div>
+            <div class="flex icons">
+                <i class="fab fa-facebook margin"></i>
+                <i class="fab fa-twitter margin"></i>
+                <i class="fas fa-envelope margin"></i>
+                <i class="fas fa-share margin"></i>
+                <i class="far fa-bookmark margin"></i>
+                <i class="far fa-comment-alt margin"></i>
+            </div>
+        </div>
+        
+            <p class="para">CAMBRIDGE, Mass. — One night late in 1979, an itinerant young physicist named Alan Guth,
+                with a new son and a year’s appointment at Stanford, stayed up late with his notebook and equations,
+                venturing far beyond the world of known physics.
+            </p>
+
+            <p class="para">He was trying to understand why there was no trace of some exotic particles that
+                should have been created in the Big Bang. Instead he discovered what might have
+                made the universe bang to begin with. A potential hitch in the presumed course of cosmic
+                evolution could have infused space itself with a special energy that exerted a repulsive force,
+                causing the universe to swell faster than the speed of light for a prodigiously violent instant.
+            </p>
+
+            <p class="para">If true, the rapid engorgement would solve paradoxes like why
+                the heavens look uniform from pole to pole and not like a jagged, warped mess.
+                The enormous ballooning would iron out all the wrinkles and irregularities.
+                Those particles were not missing, but would be diluted beyond detection, like spit in the ocean.
+            </p>
+
+            <p class="para">Reaching back across 13.8 billion years to <a href="#">the first sliver of cosmic time</a>
+                with telescopes at the South Pole, a team of astronomers led by John M. Kovac of the Harvard-Smithsonian
+                Center for
+                Astrophysics detected ripples in the fabric of space-time —
+                so-called <a href="#">gravitational waves</a> — the signature of a universe being wrenched
+                violently apart when it was roughly a trillionth of a trillionth of a trillionth
+                of a second old. They are the long-sought smoking-gun evidence of inflation, proof,
+                Dr. Kovac and his colleagues say, that Dr. Guth was correct.
+            </p>
+
+            <div class="mid">
+                <hr>
+                <h3>Thanks for reading The Times.</h3>
+                <a class="blue" href="#">Subscribe to The Times</a>
+                <hr>
+            </div>
```

Comment:
- [ ]  Please center the elements to have the same UI with the original page( the content is aligned left in your project):

![image](https://user-images.githubusercontent.com/60184961/90810097-1d20a900-e32b-11ea-8e61-c297afc67991.png)